renderer/containers/editor.js

```diff
@@ -158,7 +158,7 @@ export default class EditorContainer extends Container {
         fps,
         startTime,
         endTime,
-        muted
+        muted: isMuted
```

Comment:
I think `muted` should be renamed to `isMuted`, both here and on the main process side. Better to be consistent.

Reply:
eeae224